docs/volunteering/writing/editing-bobadocs.md

```diff
@@ -0,0 +1,158 @@
+
+Once there, on the top right corner of the GitHub page, you should see a `Fork` button. Click it. You will be automatically redirected to your fork.
+
+![a cropped screenshot of the bobadocs github page, with a red arrow pointing to the fork button](/img/volunteer/bobadocsfork.jpg)
+
+### Branching
+
+In git, a `branch` is a separate version of a repo that can be edited without affecting the main repo. It lets developers separate what they're working on, so problems that arise when working on one issue don't affect any other work that may be being done concurrently. Making good use of branches is key to keeping a project history easy to understand. It also keeps multiple people working on the same project from stepping on each other's toes.
+
+To create a new branch, click the dropdown labelled "main" on the left side of the screen, enter a short, descriptive name for your repo in the textbox, and click "Create branch".
+
+![a cropped screenshot of the bobadocs github page, with red arrows pointing to relevant spots on the branch dropdown](/img/volunteer/bobadocsbranch.jpg)
+
+## Step 1: Navigating to GitHub's editor
+
+### Method 1: Navigate from within your GitHub repo
+
+If you look at your repo, you'll notice that it looks a lot like a filesystem. That's because it is a filesystem! And you can navigate to the different files in it - that is, the *pages in the site* - by clicking through it.
+
+To do that, we need to know a little bit about how Docusaurus is laid out. Docusaurus separates site content into four folders:
+
+- `blog`: containing blogposts (this is unused by BobaDocs at this time)
+- `docs`: containing the pages on the site **(this is the one we want!)**
+- `src`: containing information on how to build the site, including styling
+- `static`: containing static site data, such as images
+
+We want to edit a page in the site, so click `docs`.
+
+![a screenshot of the git repo, with a red arrow pointing to the docs folder](/img/volunteer/bobadocsnav1.jpg)
+
+You'll notice four new folders: `engineering`, `product`, `users`, and `volunteer`. Each of these folders corresponds to one of the guides on BobaDocs. Click the folder for the guide with the page you want to edit.
+
+Once there, you'll notice something a little odd:
+
+![a screenshot of the git repo compared to the sidebar on BobaDocs. lines are drawn between items with similar names](/img/volunteer/bobadocsnav2.jpg)
+
+Some of the names of these things are QUITE SIMILAR. This isn't a coincidence! Docusaurus generates the sidebar for BobaDocs based on the files and folders present: each file is a page on the site, while each folder is a section on the sidebar.
+
+Armed with this information (and a little investigation), I determine that the page I want to edit, Codebase Overview, is named `intro.md` in the repo. I click it.
```

Comment:
Did you mention beforehand which page you want to edit?

I would split these sections:

#### Understanding the repo structure

[previous explanation]

#### Finding your file

For the purpose of this guide, we'll edit the Codebase Overview page [link, potentially screenshot of the parts that are helpful].

You could point out that the structure of the folder is actually the same as the url:
> https://bobadocs.netlify.app/**docs/engineering/intro**

I would also point out the steps a bit more:

1 - Codebase Overview is not in a menu, so it has to be one of the files
2 - clicking around, I open intro.md and confirm that the title at the top of the page is the correct one

With that said, I should probably rename intro codebase-overview and make that a best practice TBH.